8.x/theme-development/basics.md

```diff
@@ -2,4 +2,190 @@
 
 [[toc]]
 
-Should be filled.
\ No newline at end of file
+In this tutorial, the basics of the Larammerce Template Engine will be explained. As you know, Larammerce is based on Laravel, and if you check a Larammerce template, you see that Larammerce blade files are the same as Laravel blade files. In the previous tutorial, a blade page was created, but in this tutorial, the general rules will be explained and then the template will be developed.
+
+.blade.php files are in the following directory `public/views/example.blade.php`.
+
+There are three types of blad files in this template engine.
+
+![blade-files](/blade-files-1-2.png)
+
+1- File name starts with underscore (partial files)
+
+2- File name does not start with underscore
+
+NOTE: A blade partial is similar to include or require in PHP. It’s an easy way to include contents of another file inside of a template.
+
+For example,  using the following commands extends the **_base.blade.php** page to other pages:
+```php
+@extends('_base') # Write this command in the about.blade.php file.
+```
+And to use the  **_meta_tags.blade.php** page on other pages using the following commands:
+```php
+@section('meta_tags')
+    @include('_meta_tags', ['obj' => $web_page])
+    <meta property="og:type" content="website">
+@endsection # Write this command in the index.blade.php file.
+```
+So to create a layout that extends to all pages, put an underscore at the beginning of the page's name.
+
+3- The end of the file name is **_mobile** or **_app**.
+
+![blade-file-3](/blade-file-3.png)
+
+As you know, responsive web design works through CSS, using various settings to different style properties depending on the screen size of the user’s device. Sometimes the design structure of pages on mobile and desktop is different, so create a file with the **_mobile** extension to be displayed on mobile pages with this structure. And for mobile applications, the system provides **_app** pages.
+
+Blade pages are reviewed below. Some pages are required and should be in the template, but some pages are optional.
+
+## About page
+![about](/about.png)
```

Comment:
There is no need to add a section for about or any other optional files, at the end of the document please add a section and mention that if you want a custom web page with a specific web address (URL) please add your blade file like this and do the following to see a preview of your page.